src/NBitcoin.Tests/ChainTests.cs

```diff
@@ -170,21 +170,76 @@ public void CanBuildChain()
         [Trait("UnitTest", "UnitTest")]
         public void CanFindFork()
         {
-            var chain = new ConcurrentChain(this.network);
-            var chain2 = new ConcurrentChain(this.network);
-
-            this.AppendBlock(chain);
-            ChainedHeader fork = this.AppendBlock(chain);
-            ChainedHeader tip = this.AppendBlock(chain);
-
-            this.AssertFork(chain, chain2, chain.Genesis);
-            chain2 = new ConcurrentChain(this.networkTest);
-            this.AssertFork(chain, chain2, null);
-            chain2 = new ConcurrentChain(this.network);
-            chain2.SetTip(fork);
-            this.AssertFork(chain, chain2, fork);
-            chain2.SetTip(tip);
-            this.AssertFork(chain, chain2, tip);
+            var chain1 = new ConcurrentChain(this.network);
+
+				//create the main chain with a (commonChainSize) blocks before the fork
+				int commonChainSize = 150000;
+				for (int i = 0; i < commonChainSize; i++) {
+					this.AppendBlock(chain1);
+				}
+
+				ChainedHeader fork = this.AppendBlock(chain1);
+				//add some blocks from the fork point to the tip
+				int chain1ForkDepth = 10;
+				for (int i = 0; i < chain1ForkDepth; i++) {
+					this.AppendBlock(chain1);
+				}
+				//tip of the main chain
+				ChainedHeader tip = chain1.Tip;// this.AppendBlock(chain);
```

Comment:
Generally we avoid introducing commented out code to the codebase

Reply:
oversight